### ISSUE TITLE
Infinite loop when getting config file

### ISSUE THREAD

**Issue by pavlovicIvan** [HashLips/hashlips_nft_minting_dapp#2]
Hello, noticed in the network tab that config.js GET API is called infinite times, and app after some period is getting unresponsive.

Maybe adding empty dependency array at the end of useEffect will help?

```
  useEffect(() => {
    getConfig();
  }, []);
```

**Comment by MultiplierFinance1** [comment]
yeah i noticed the same thing. Wanted to post this too. then I saw you already did👍 I hope he will see this and fix it in the code!
